content/en/docs/Configuration/Certificate Management/mtls.md

```diff
@@ -8,35 +8,37 @@ date: 2021-09-02
 
 ## Context
 
-Mutual authentication is required for most of API developed for Open Banking.
+Mutual authentication is required for most APIs developed for Open Banking.
 
-According to the Open Banking Specification, the MTLS is required for both components : Cloud Entity and the API Gateway Listener. Here is a diagram that explains how it's working.
+According to the Open Banking Specification, the MTLS is required for the Cloud Entity and API Gateway Listener components. See the diagram which explains how it is working.
 ![MTLS diagram](/Images/mtls.png)
```

Comment:
According to the Open Banking Specification, Mutual Transport Layer Security (MTLS) based client connection is required for the Cloud Entity and API Gateway Listener components. See the diagram for details on the MTLS setup.